doc/bootloader-building.rst

```diff
@@ -15,7 +15,7 @@ or if you want to modify the |bootloader| source,
 you need to build the |bootloader|.
 To do this,
 
-* Download and install Python, which is required for running `:command:waf`,
+* Download and install Python, which is required for running ``waf``,
```

Comment:
This should be 
```
:command:`waf`
``` 

per https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-command.